src/Components/App.jsx

```diff
@@ -28,6 +31,9 @@ function App() {
         <Route path="/users" element={<Users />} />
         <Route path="*" element={<NoPage />} />
         <Route path="/article/:id" element={<SingleArticle />} />
+        <Route path="/article/topic/football" element={<FootballPage />} />
```

Comment:
Instead of hardcoding all paths, I recommend using a parametric path and the 'useParams' hook to make it dynamic.